library/Icingadb/Widget/Detail/HostDetail.php

```diff
@@ -25,8 +25,7 @@ public function __construct(Host $object, ServicestateSummary $serviceSummary)
     protected function createServiceStatistics(): array
     {
         if ($this->serviceSummary->services_total > 0) {
-            $services = new ServiceStatistics($this->serviceSummary);
-            $services->setBaseFilter(Filter::equal('host.name', $this->object->name));
+            $services = new ServiceStatistics($this->serviceSummary, $this->object);
```

Comment:
Please restore the call to `setBaseFilter()` and call the `setUrl()` on `ServiceStatistics` to override the URL instead.